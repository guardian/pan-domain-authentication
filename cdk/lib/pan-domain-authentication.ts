import type { GuStackProps } from "@guardian/cdk/lib/constructs/core";
import { GuStack } from "@guardian/cdk/lib/constructs/core";
import type { App } from "aws-cdk-lib";

export class PanDomainAuthentication extends GuStack {
  constructor(scope: App, id: string, props: GuStackProps) {
    super(scope, id, props);
  }
}
