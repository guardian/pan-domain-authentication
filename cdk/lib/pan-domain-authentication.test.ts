import { App } from "aws-cdk-lib";
import { Template } from "aws-cdk-lib/assertions";
import { PanDomainAuthentication } from "./pan-domain-authentication";

describe("The PanDomainAuthentication stack", () => {
  it("matches the snapshot", () => {
    const app = new App();
    const stack = new PanDomainAuthentication(app, "PanDomainAuthentication", { stack: "pan-domain-authentication", stage: "TEST" });
    const template = Template.fromStack(stack);
    expect(template.toJSON()).toMatchSnapshot();
  });
});
