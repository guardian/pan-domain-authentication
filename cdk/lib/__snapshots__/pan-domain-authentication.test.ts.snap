// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The PanDomainAuthentication stack matches the snapshot 1`] = `
{
  "Metadata": {
    "gu:cdk:constructs": [],
    "gu:cdk:version": "TEST",
  },
}
`;
